SerialPrograms/Source/CommonFramework/ImageMatch/WaterfillTemplateMatcher.cpp

```diff
@@ -82,6 +82,15 @@ bool WaterfillTemplateMatcher::check_aspect_ratio(size_t candidate_width, size_t
              << " (" << (double)candidate_width/candidate_height << ") "
              << "error: " << error << " bound [" << m_aspect_ratio_lower << ", " << m_aspect_ratio_upper << "]" << endl;
     }
+    else if (pass && PreloadSettings::debug().IMAGE_TEMPLATE_MATCHING){

```

Comment:
Since both halves of this if-statement require `PreloadSettings::debug().IMAGE_TEMPLATE_MATCHING`. Might as well just wrap the entire thing with an if-statement for that, then branch it into pass = true or false.

Reply:
Fixed. I also did more refactoring of the debugging print statements.